tensorrt_llm/inputs/registry.py

```
@@ -712,17 +712,37 @@ def create_input_processor(
     config = None
 
     if checkpoint_format == "HF":
         try:
             model_config = ModelConfig.from_pretrained(model_path_or_dir,
                                                        trust_remote_code=True)
             config = model_config.pretrained_config
-        except (ValueError, EnvironmentError) as e:
-            logger.debug(
-                f"Unable to load HF config from {model_path_or_dir}: {e}. Falling back."
-            )
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)